cmd/pmm-managed-starlark/main.go

```diff
@@ -46,4 +63,63 @@ func main() {
 	if on, _ := strconv.ParseBool(os.Getenv("PMM_TRACE")); on {
 		logrus.SetLevel(logrus.TraceLevel)
 	}
+	if on, _ := strconv.ParseBool(os.Getenv(starlarkRecursionFlag)); on {
+		resolve.AllowRecursion = true
+	}
+
+	l := logrus.WithField("component", "pmm-managed-starlark")
+
+	decoder := json.NewDecoder(os.Stdin)
+	var data checks.StarlarkScriptData
+	err := decoder.Decode(&data)
+	if err != nil {
+		l.Errorf("Error decoding json data: %s", err)
+		os.Exit(1)
+	}
+
+	results, err := runChecks(l, &data)
+	if err != nil {
+		l.Errorf("Error running starlark script: %s", err)
+		os.Exit(1)
+	}
+
+	encoder := json.NewEncoder(os.Stdout)
+	err = encoder.Encode(results)
```

Comment:
Please add a test for that case: stdout (contains valid result JSON), stderr (empty), exit code (0)